vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -678,6 +713,24 @@ The absence of `RestrictedProperties` denotes all properties are accessible usin
           <Annotation Term="Core.IsLanguageDependent" />
         </Property>
       </ComplexType>
+      <ComplexType Name="CollectionUpdateType">

```

Comment:
Flatten this, move these four properties to UpdateRestrictionsType, mention in Description that they only are valid if applied to a collection